src/box/tuple.h

```diff
@@ -170,6 +170,29 @@ box_tuple_format(box_tuple_t *tuple);
 const char *
 box_tuple_field(box_tuple_t *tuple, uint32_t fieldno);
 
+/**
+ * Return a raw tuple field in the MsgPack format pointed by
+ * a JSON path.
+ *
+ * The JSON path includes the outmost field. For example, "c" in
+ * ["a", ["b", "c"], "d"] can be accessed using "[2][2]" path (if
+ * index_base is 1, as in Lua). If index_base is set to 0, the
+ * same field will be pointed by the "[1][1]" path.
+ *
+ * The first JSON path token may be a field name if the tuple
+ * has associated format with named fields. A field of a nested
+ * map can be accessed in the same way: "foo.bar" or ".foo.bar".
+ *
+ * The buffer is valid until a next call to a box_tuple_*
+ * function.
+ *
+ * Return NULL if the field does not exist or if the JSON path is
+ * malformed or invalid. Multikey JSON path token [*] is treated
+ * as invalid in this context.
```

Comment:
Other public functions use Doxygen-style comments for describing function arguments and return value. Shouldn't this one too?

Reply:
I'm not a big fun of Doxygen style commenting and I often see that a free form description plus the declaration itself do the good job. There are many useless lines like `@param tuple a tuple`, which are imposed by the Doxygen style commenting.

But consistency is the good point. I'll add it for consistency and will think about stripping useless comments later.